core/templates/pages/learner-dashboard-page/card-display/card-display.component.ts

```diff
@@ -34,6 +34,7 @@ export class CardDisplayComponent {
   maxShifts: number = 0;
   lastShift: number = 0;
   isLanguageRTL: boolean = false;
+  toggleState: boolean = false;
```

Comment:
This variable name is not clear, is it supposed to be something such as `currentlyToggledState`? Please change it. 

Also general advice -- avoid starting variable names with verbs, for better clarity between variable names and function names.

Reply:
I understand your point. Just to clarify, toggle can function as both a noun and verb, meaning toggleState is actually a noun (noun phrase more specifically). 

If you think currentlyToggledState would make it more clearer, I'm happy to make that change.

Reply:
Yep, please make that change

Reply:
Done